storage/Devices/Encryption.h

```diff
@@ -170,7 +170,14 @@ namespace storage
 	 */
 	void set_open_options(const std::string& open_options);
 
+	/**
+	 * Get all Encryptions.
```

Comment:
Maybe mention that encryption can be stacked in multiple layers upon one another?

Also, does this return that stack, or only the _different_ encryption types (like `sort | uniq`)?